week-2/extended-project/banzai/todo-list/cli.rb

```diff
@@ -0,0 +1,35 @@
+# define global variable
+$bd_name = File.expand_path('../', __FILE__) + '/bd.txt' 
+def read_file
+  File.read($bd_name).split("\n").map { |line| line.split(" - ").map(&:strip) }
+end
+
+def save_file(tasks)
+  newTasks = tasks.map { |fil|  fil.join(' - ') }.join("\n")
+  File.write($bd_name, newTasks)
+end
+
+def write(task)
+  list_task = read_file
+  last_index = (list_task[list_task.length - 1][0]).to_i
```

Comment:
This code: `list_task[list_task.length - 1]` is the same of `list_task.last` ?

Reply:
Is `last_index` the same of the length of the `list_task - 1`

Reply:
Now I think we should use `last_index = list_task.last.first` to get the id of the last element

Reply:
The id of the last element is not always the same with the length of the `list_task` - 1 because someone can delete elements.